src/env_setup.c

```diff
@@ -49,6 +49,9 @@ t_err	ft_env_setup(t_hashtable **env_table)
 		err = ft_insert_env_shlvl(*env_table);
 	if (err != SUCCESS)
 		return (err);
+	err = ft_insert_env_pid(*env_table);
```

Comment:
err handling in its own branch as mentioned, right?

Reply:
Yep, it'll get moved into its own function